package/config.js

```diff
@@ -28,14 +28,7 @@ config.outputPath = resolve(config.public_root_path, config.public_output_path)
 // (code-splitting chunks and static assets) load from the CDN instead of a relative path.
 const getPublicPath = () => {
   const rootUrl = process.env.WEBPACKER_ASSET_HOST || '/'
-  let packPath = `${config.public_output_path}/`
-  // Add relative root prefix to pack path.
-  if (process.env.RAILS_RELATIVE_URL_ROOT) {
-    let relativeRoot = process.env.RAILS_RELATIVE_URL_ROOT
-    relativeRoot = relativeRoot.startsWith('/') ? relativeRoot.substr(1) : relativeRoot
-    packPath = `${ensureTrailingSlash(relativeRoot)}${packPath}`
-  }
-
+  const packPath = `${config.public_output_path}/`
   return ensureTrailingSlash(rootUrl) + packPath
```

Comment:
Nitpick: you might move `ensureTrailingSlash` like:
```js
const rootUrl = ensureTrailingSlash(process.env.WEBPACKER_ASSET_HOST || '/')

return `${rootUrl}${config.public_output_path}/`
```
I could even see this, but it might not read as well:
```js
return `${ensureTrailingSlash(process.env.WEBPACKER_ASSET_HOST || '/')}${config.public_output_path}/`
```

Reply:
Your first suggestion is more readable. I wasn't trying to touch more than was necessary here but I can add that change.